src/main/java/com/azure/cosmos/examples/changefeed/SampleChangeFeedProcessor.java

```diff
@@ -0,0 +1,269 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.examples.changefeed;
+
+import com.azure.cosmos.ChangeFeedProcessor;
+import com.azure.cosmos.ConnectionPolicy;
+import com.azure.cosmos.ConsistencyLevel;
+import com.azure.cosmos.CosmosAsyncClient;
+import com.azure.cosmos.CosmosAsyncContainer;
+import com.azure.cosmos.CosmosAsyncDatabase;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.CosmosClientException;
+import com.azure.cosmos.examples.common.CustomPOJO;
+import com.azure.cosmos.implementation.Utils;
+import com.azure.cosmos.models.CosmosAsyncContainerResponse;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosContainerRequestOptions;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.scheduler.Schedulers;
+
+import java.time.Duration;
+import java.util.List;
+
+/**
+ * Sample for Change Feed Processor.
+ */
+public class SampleChangeFeedProcessor {
+
+    public static int WAIT_FOR_WORK = 60000;
+    public static final String DATABASE_NAME = "db_" + RandomStringUtils.randomAlphabetic(7);
+    public static final String COLLECTION_NAME = "coll_" + RandomStringUtils.randomAlphabetic(7);
+    private static final ObjectMapper OBJECT_MAPPER = Utils.getSimpleObjectMapper();
+    protected static Logger logger = LoggerFactory.getLogger(SampleChangeFeedProcessor.class.getSimpleName());
+
+
+    private static ChangeFeedProcessor changeFeedProcessorInstance;
+    private static boolean isWorkCompleted = false;
+
+    public static void main(String[] args) {
+        logger.info("BEGIN Sample");
+
+        try {
+            //This sample models an application where documents are being inserted into one container (the "feed container"),
+            //and meanwhile another worker thread or worker application is pulling inserted documents from the feed container's Change Feed
+            //and operating on them in some way. For one or more workers to process the Change Feed of a container, the workers must first contact the server
+            //and "lease" access to monitor one or more partitions of the feed container. The Change Feed Processor Library
+            //handles leasing automatically for you, however you must create a separate "lease container" where the Change Feed
+            //Processor Library can store and track leases container partitions.
+
+            //Summary of the next four commands:
+            //-Create an asynchronous Azure Cosmos DB client and database so that we can issue async requests to the DB
```

Comment:
We need to add a disclaimer here that by creating the Cosmos containers and executing work on them, there will be charges associated with. We should also capture the default RU we set and link to the charges doc/web page.